cirq-core/cirq/ops/clifford_gate_test.py

```diff
@@ -739,7 +740,7 @@ def test_multi_clifford_decompose_by_unitary():
     # Construct a random clifford gate:
     n, num_ops = 5, 20  # because we relied on unitary cannot test large-scale qubits
     gate_candidate = [cirq.X, cirq.Y, cirq.Z, cirq.H, cirq.S, cirq.CNOT, cirq.CZ]
-    for seed in range(100):
+    for seed in range(10):
```

Comment:
I'd suggest to drop seeding here and replace it with np.random calls and a simple repetition.
The default numpy random generator is getting reproducibly seeded
in [check/pytest](https://github.com/pavoljuhas/Cirq/blob/d55a99a20b996b4afc3cc23822879c026fe3b37d/check/pytest#L44-L45).  Over time there should be more randomness in the generated choices than with the 10 seeds here.

Reply:
Good call. Done

Reply:
Nice!